### PR TITLE
Added wallet info

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/shared/forms/budgets-form/dtos.ts [hunk 1]
```diff
@@ -11,36 +11,44 @@ export interface BudgetDto {
...
     name: 'Pet',
+    icon: 'fas fa-fish',
     isSelected: false
   },
   {
     name: 'Charity',
+    icon: 'fas fa-hand-holding-heart',
     isSelected: false
   }
 ];
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/budgets-form/dtos.ts:47]: Try to generate 'isSelected' initial value with iterator function, like map.

File: src/app/home/shared/forms/income-form/income-form.component.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="container" @fade>
   <div class="main-icon-container">
-    <i class="fas fa-money-bill-wave main-icon"></i>
+    <img src="assets/images/main-icon.svg" alt="image" class="main-icon">
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/income-form/income-form.component.html:3]: The purpose of `alt` is to describe what image is that, for better accessibility. I'm afraid, 'image' might be a little bit generic. Just a little bit. Generic. A little.

File: src/app/home/shared/forms/wallet-form/wallet-form.component.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="container" @fade>
   <div class="main-icon-container">
-    <i class="fas fa-money-bill-wave main-icon"></i>
+    <img src="assets/images/main-icon.svg" alt="image" class="main-icon">
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/wallet-form/wallet-form.component.html:3]: Another `alt` that might be better 🐍

File: src/app/shared/components/card-slider/card-slider.component.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  align-items: center;
+}
+
+.wallet-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 240px;
+  height: 140px;
+  border-radius: 20px;
```
Comment (FilipTalaga) [inline src/app/shared/components/card-slider/card-slider.component.scss:34]: REMs? 👯‍♀️

File: src/app/shared/components/card-slider/card-slider.component.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  @Input() walletList: WalletDto[];
+
+  active = 0;
+  isSwipingLeft = false;
+  isSwipingRight = true;
+
+  switchCardOnRight() {
+    this.isSwipingLeft = false;
+    this.isSwipingRight = true;
+    this.active = this.active < this.walletList.length - 1 ? this.active + 1 : this.active;
```
Comment (FilipTalaga) [inline src/app/shared/components/card-slider/card-slider.component.ts:22]: Maybe not, but maybe? :D

`this.active += this.active < this.walletList.length - 1`

File: src/app/shared/components/card-slider/card-slider.component.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  switchCardOnLeft() {
+    this.isSwipingRight = false;
+    this.isSwipingLeft = true;
+    this.active = this.active > 0 ? this.active - 1 : this.active;
```
Comment (FilipTalaga) [inline src/app/shared/components/card-slider/card-slider.component.ts:28]: This could also be:

`this.active -= this.active > 0`

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**